src/ucs/debug/instrument.c

```diff
@@ -133,7 +196,7 @@ void __ucs_instrument_record(uint64_t location, uint64_t lparam, uint32_t wparam
     current->timestamp = ucs_get_time();
     current->lparam    = lparam;
     current->wparam    = wparam;
-    current->location  = (uint32_t)location; /* chop off high dword */
+    current->location  = location;
```

Comment:
this commit breaks ucx_read_instr because the location now is an id instead of address. pls fix it.